tests/Doctrine/Tests/DBAL/Platforms/AbstractPlatformTestCase.php

```diff
@@ -1429,4 +1429,18 @@ public function getGeneratesFloatDeclarationSQL()
             array(array('precision' => 8, 'scale' => 2), 'DOUBLE PRECISION'),
         );
     }
+
+    public function testFixGithub2538Collation()
```

Comment:
Please use a method name describing the issue being covered. Also, use `@group #2538` for labeling it